DominicanWhoCodes.Blazor/Pages/Index.razor

```diff
@@ -33,5 +34,8 @@
         public string Image { get; set; }
         public string Summary { get; set; }
         public string Skills { get; set; }
+        public string Wegpage { get; set; }
+        public string Linkedin { get; set; }
```

Comment:
This is a compose work, should be `LinkedIn`

Reply:
I need a nacho book